addons/account/report/account_aged_partner_balance.py

```diff
@@ -52,23 +52,18 @@ def _get_partner_move_lines(self, account_type, date_from, target_move, period_l
         move_state = ['draft', 'posted']
         if target_move == 'posted':
             move_state = ['posted']
-        arg_list = (tuple(move_state), tuple(account_type))
-        #build the reconciliation clause to see what partner needs to be printed
-        reconciliation_clause = '(l.reconciled IS FALSE)'
-        cr.execute('SELECT debit_move_id, credit_move_id FROM account_partial_reconcile where max_date > %s', (date_from,))
-        reconciled_after_date = []
-        for row in cr.fetchall():
-            reconciled_after_date += [row[0], row[1]]
-        if reconciled_after_date:
-            reconciliation_clause = '(l.reconciled IS FALSE OR l.id IN %s)'
-            arg_list += (tuple(reconciled_after_date),)
+        arg_list = (tuple(move_state), tuple(account_type), date_from, date_from,)
         if ctx.get('partner_ids'):
             partner_clause = 'AND (l.partner_id IN %s)'
             arg_list += (tuple(ctx['partner_ids'].ids),)
         if ctx.get('partner_categories'):
-            partner_clause += 'AND (l.partner_id IN %s)'
-            partner_ids = self.env['res.partner'].search([('category_id', 'in', ctx['partner_categories'].ids)]).ids
```

Comment:
As far as I can see, this is the major difference with #41385 . What is the performance impact of this part of the query? Is it really necessary to change it?

Reply:
Thanks for your review.
I did some testing in v12, having 7000 res.partner in one category_id, or using a few different category_id, but did not see a performance difference, so I removed these changes.